repos/system_upgrade/el7toel8/actors/spamassassinconfigread/libraries/library.py

```diff
@@ -0,0 +1,35 @@
+from leapp.libraries.actor import lib_spamc, lib_spamd
+from leapp.libraries.common.rpms import has_package
+from leapp.libraries.stdlib import api
+from leapp.models import InstalledRedHatSignedRPM, SpamassassinFacts
+
+
+def read_file(path):
```

Comment:
what about add read_file function into the common libraries? (utils?) This is pretty common function that is used on many places. I am ok to keep it here, but thinking about refactoring in future as it looks like duplicit stuff to define the same function on many places.

Reply:
consider actors should by Py2/Py3 compatible. It's possible we will switch to Py3 in future (even on RHEL 7) regarding the Py2 EOL.

Reply:
I moved it to utils. To the best of my knowledge, it is python3 compatible.